### PR TITLE
5_1c to 5_2_2

### PR DESCRIPTION
**Pull request by nolo16**

### CODE REVIEW

File: script/Arcueid/Day 5/05_00_ARC05_1C.txt [hunk 1]
```diff
@@ -1,17 +1,17 @@
 [sha:ef927d7cb71016e9d6cbdb44591afc6fcaf0d1c5]{
 -- Page 10002, Offset 8930.
 -- とは言っても、こんなのは俺だって初体験だ。
-But even if I say that, this is really a first for me.
+Well, that being said, it's not like I've ever dealt with a situation like this before.
```
Comment (yk1512) [inline script/Arcueid/Day 5/05_00_ARC05_1C.txt:4]: remove the well

File: script/Arcueid/Day 5/05_00_ARC05_1C.txt [hunk 2]
```diff
@@ -21,12 +21,12 @@ Fortunately, I have a ton of money.
 [sha:79a85382b633ab382fd47156e46ea43ca1be4dac]{
 -- Page 144, Offset 8934.
 -- 素人考えだろうと、何も無いよりはあった方がいい。何があいつの助けになるか分からないんだから。
-Even if it's an amateurish idea, it's better than coming up empty-handed. You never know what might be of help to that woman.
+Although I'm no expert, my amateur opinion is definitely better than coming up empty-handed. You never know what might be of help to her.
```
Comment (yk1512) [inline script/Arcueid/Day 5/05_00_ARC05_1C.txt:24]: Prefer the original for the first sentence.

File: script/Arcueid/Day 5/05_00_ARC05_1C.txt [hunk 3]
```diff
@@ -66,23 +66,23 @@ Or something like that.
 [sha:15593bc9a15f5b80d971a3b59fa217c1f90647ad]{
 -- Page 148, Offset 8943.
 -- どうやら内臓はそのうち勝手に治るらしい。あまりに常識外なのでそのあたりは話題にしないよう聞き流す。
-In any case, it seems her internal organs will heal on their own in no time. This is so out of the ordinary that I don't even want to question it.
+Anyway, it seems like her internal organs will heal themselves soon. It's so outlandish that I'm not even going to bother questioning it.
```
Comment (yk1512) [inline script/Arcueid/Day 5/05_00_ARC05_1C.txt:69]: prefer 'on their own'

**Comment by yk1512** [comment]
Only checked up to Offset 8943. Don't know if I'll have time to finish or not though so don't wait on me to merge

File: script/Arcueid/Day 5/05_00_ARC05_2.txt [hunk 4]
```diff
@@ -61,12 +61,12 @@ But we do know each other's names and identities. It's only natural that the mor
 [sha:7d0513928450730fbdced2db00963683f72824b9]{
 -- Page 179, Offset 9019.
 -- 「んー、まあ、吸血鬼たちの王さまみたいなもの、かな。
-"Hmm, well, it's kind of like a king among vampires, I guess.
+"Hmm, well, something like a king among vampires, I guess.
```
Comment (yk1512) [inline script/Arcueid/Day 5/05_00_ARC05_2.txt:64]: they're like a king among vampires, I guess.

File: script/Arcueid/Day 5/05_00_ARC05_2.txt [hunk 5]
```diff
@@ -332,27 +332,27 @@ This is a society of humans. A city of humans.
...
 -- Page 196, Offset 9073.
 -- 人間が弱すぎたから、吸血鬼が強すぎたから、という話じゃない。
-This is not a matter of humans being too weak or vampires being too strong.
+This isn't a matter of humans being too weak or vampires being too strong.
 }
 [sha:78ceb1cbbdb41a6faa1c8f1146c97095f78a3aa1]{
 -- Page 197, Offset 9074.
 -- 都市とは単体では生きづらいから、集団になって個々を守る人間の防衛機能だ。
-People form cities because it is difficult to survive on your own; they are a defense mechanism to protect the individual through the power of the collective.
+People form cities because it's too difficult to survive on your own; they are a defense mechanism to protect the individual through the power of the collective.
```
Comment (yk1512) [inline script/Arcueid/Day 5/05_00_ARC05_2.txt:340]: too difficult for them to survive on their own

File: script/Arcueid/Day 5/05_00_ARC05_2_2.txt [hunk 6]
```diff
@@ -167,22 +167,22 @@ But still――― Even a person like me can say what I don't want to do.
...
 -- Page 213, Offset 9123.
 -- 「……アルクェイド。おまえの話は、俺には分からない。
-"...Arcueid. I don't understand what you're talking about.
+"...Arcueid. I don't get what you're talking about.
 }
 [sha:17b851d0690052eb46aaf68ed7d47c5d7e404ebd]{
 -- Page 213, Offset 9124.
 -- どんなに相手が違う生き物だとしても、どんなに俺が無力だとしても、責任は同じなんだ。俺はやっぱり、多くの人を見殺しにしたし、多くの人を見殺しにするのはイヤなんだ。
-No matter how much of a monster he is, no matter how powerless I am, my responsibility is still the same. As I thought, I let many people die, and I don't want to see anyone else meet that fate.
+No matter how much of a monster he is, no matter how powerless I am, my responsibility is still the same. Just as I thought, I let many people die, and I don't want to see anyone else meet that fate.
```
Comment (yk1512) [inline script/Arcueid/Day 5/05_00_ARC05_2_2.txt:175]: Rather than 'just as I thought' I think something like 'Regardless of how you put it,' fits better

**Review by yk1512 — APPROVED** [review]

File: script/Arcueid/Day 5/05_00_ARC05_1C.txt [hunk 7]
```diff
@@ -1,17 +1,17 @@
...
 -- Page 10002, Offset 8930.
 -- とは言っても、こんなのは俺だって初体験だ。
-But even if I say that, this is really a first for me.
+Well, that being said, it's not like I've ever dealt with a situation like this before.
 }
 [sha:9a706387e658400203894f78ed1c3145d64f96a8]{
 -- Page 10002, Offset 8931.
 -- 応急手当とは言ったものの、思いつくのは、消毒薬、傷口を塞ぐガーゼ、包帯、鎮痛剤ぐらい。
-The only first aid stuff I can think of is gauze for covering the wound, antiseptic, bandages, and painkillers.
+The only first aid measures that come to mind are gauze for covering the wound, antiseptic, bandages, and painkillers.
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_1C.txt:9]: covering her wound

File: script/Arcueid/Day 5/05_00_ARC05_1C.txt (hunk 2 above)
Comment (Avuxo) [reply script/Arcueid/Day 5/05_00_ARC05_1C.txt:24]: I prefer new version

File: script/Arcueid/Day 5/05_00_ARC05_1C.txt [hunk 8]
```diff
@@ -66,23 +66,23 @@ Or something like that.
...
 -- どうやら内臓はそのうち勝手に治るらしい。あまりに常識外なのでそのあたりは話題にしないよう聞き流す。
-In any case, it seems her internal organs will heal on their own in no time. This is so out of the ordinary that I don't even want to question it.
+Anyway, it seems like her internal organs will heal themselves soon. It's so outlandish that I'm not even going to bother questioning it.
 }
 [sha:7d6d01510a5129d467be22cc628b4a35a771d701]{
 -- Page 148, Offset 8944.
 -- 出血は止まってあとは傷口を塞ぐだけなのだが、そこから先が思うようにいかないのだとか。
 // emphasis on human mine. そこから先が思うようにいかないのだとか lit. From here, things won't go as I expect referring to Arcueid's toughness.
-The bleeding has stopped and all that's left to do is to close the wound, but from this point on, things are different from what I might expect from a human.
+The bleeding's stopped, and all that's left is to close the wound, but from this point on, I doubt things are going to play out as they would with a human.
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_1C.txt:75]: wound. But from

File: script/Arcueid/Day 5/05_00_ARC05_1C.txt [hunk 9]
```diff
@@ -92,7 +92,7 @@ I was afraid that she would go crazy from being soaked, but she doesn't budge at
 [sha:679fadf71c57671f54e2a572e837f811ba81e20a]{
 -- Page 149, Offset 8948.
 -- という鈍感……いや、頑丈さである。
-She doesn't seem very sensitive... no, rather she's tough.
+She doesn't seem very sensitive... no, she's tough.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Arcueid/Day 5/05_00_ARC05_1C.txt:95]: just revert imo

File: script/Arcueid/Day 5/05_00_ARC05_1C.txt [hunk 10]
```diff
@@ -117,7 +117,7 @@ Arcueid lets out a cheerful laugh, seemingly amused by something.
 [sha:cf92145e1da7b7a3c6acbdc7d318fe563a83d182]{
 -- Page 151, Offset 8953.
 -- 消毒薬ではびくともしなかったクセに、包帯はくすぐったいらしい。
-Despite being unresponsive to the antiseptic, it appears that the bandage is able to tickle her.
+Despite being unresponsive to the antiseptic, it seems like she thinks the bandage is ticklish.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Arcueid/Day 5/05_00_ARC05_1C.txt:120]: neither are quite right and make it sound like the bandage itself is ticklish.
> it seems like the bandage is tickling her.

File: script/Arcueid/Day 5/05_00_ARC05_1C.txt [hunk 11]
```diff
@@ -187,7 +188,7 @@ I move away from Arcueid and sit down against the wall.
 [sha:87927de873c3a1a78a26b9d86073ae0fcfaa8c53]{
 -- Page 157, Offset 8967.
 -- 眠って力が戻ればすぐに治るんだろ？　見張っててやるから、おとなしく寝ていてくれ」
-You'll get better as soon as you sleep and recover your strength, right? I'll keep an eye on you, so just be quiet and go to sleep."
+You'll get better faster if you sleep and recover your strength, right? I'll stand guard, so you can just sleep tight."
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_1C.txt:191]: prefer "be quiet and go to sleep" for おとなしく寝て

File: script/Arcueid/Day 5/05_00_ARC05_1C.txt [hunk 12]
```diff
@@ -242,12 +243,12 @@ She's in full-on chatter mode.
 [sha:d4036d48f76d9bbf37d857130ebb1a7ea35edfb7]{
 -- Page 161, Offset 8978.
 -- 仕方ない。
-I don't have a choice.
+Guess there's no choice.
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_1C.txt:246]: Guess I've got no choice.

File: script/Arcueid/Day 5/05_00_ARC05_1C.txt [hunk 13]
```diff
@@ -323,12 +324,12 @@ If you were to compare my Lifescale with Vlov's right now, he would be slightly
 [sha:e8f11727089cfd7149d6876d813674b4ae448da2]{
 -- Page 168, Offset 8994.
 -- 普段のアルクェイドなら、なんとか倒せる相手なんだ？」
-So if you were in your normal condition, you would be able to defeat him somehow, right?"
+So if you were at peak performance, you'd be able to defeat him somehow, right?"
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_1C.txt:327]: revert "normal condition"

File: script/Arcueid/Day 5/05_00_ARC05_2_2.txt [hunk 14]
```diff
@@ -142,7 +142,7 @@ The lines of death that only I can see.
 [sha:e5232a3a77cfe256c4b3120435702133d4a85365]{
 -- Page 211, Offset 9118.
 -- この両眼は、本当に必要になる時があるから備わったと、大切な人に教えられた。
-I was told by someone I hold dear that I was endowed with these eyes for times of great necessity.
+I was told by someone I hold dear that I was bestowed these eyes for times of great necessity.
```
Comment (Avuxo) [inline script/Arcueid/Day 5/05_00_ARC05_2_2.txt:145]: revert

**Review by Avuxo — APPROVED** [review]
otskuare

File: script/Arcueid/Day 5/05_00_ARC05_1C.txt [hunk 15]
```diff
@@ -267,12 +268,12 @@ I've got some questions I need to ask her as well, so I'll just talk with her fo
...
 -- Page 163, Offset 8983.
 -- 吸血鬼なのは分かる。仰々しい服とか、おまえよりよっぽどらしかったからな。けど―――」
-I get that he's a vampire, I mean he's got those elaborate clothes on and all, so he looks way more vampire-like than you at least."
+I get that he's a vampire. I mean, he's got those elaborate clothes on and all, so he looks way more vampire-like than you at least."
 }
 [sha:ad8345166d09815cbec61df3f15d27036ccd5980]{
 -- Page 164, Offset 8984.
 -- あの青色の炎。廊下に充満し、人間だけを燃やすあの火はなんなのか。あれは吸血鬼のイメージとはかけ離れすぎている。
-Those blue flames... spreading through the hallway, burning only humans in their wake... What are they? That's not how I imagine vampires are like at all.
+Those blue flames. Those flames that raged throughout the hallway, only burning humans in their wake. Just what are they? They don't fit with my image of a vampire at all.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 5/05_00_ARC05_1C.txt:276]: I feel like I left this comment somewhere else, but 'burning in their wake' is kinda weird, the wake is where the _aftermath_ is. 
> Those blue flames... What on earth were they? They swept through the corridor but they only actually burned people. They don't fit with...

File: script/Arcueid/Day 5/05_00_ARC05_1C.txt (hunk 13 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 5/05_00_ARC05_1C.txt:327]: You may not like it but this is what normal condition looks like

File: script/Arcueid/Day 5/05_00_ARC05_2.txt [hunk 16]
```diff
@@ -181,7 +181,7 @@ Then that hotel tragedy...
 [sha:578ea89ba499e2419f54076694aef6c3102db2ae]{
 -- Page 187, Offset 9043.
 -- ホテルの人間を殺したのはヴローヴでも、その災害を呼びこんだのは自分だと。
-Although Vlov was the one that killed those people, we were the ones that called down that disaster.
+Although Vlov was the one that killed those people, we were the ones that brought forth that disaster.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_2.txt:184]: I think I prefer called down, brought forth suggests they had just a slightly more active hand in it

File: script/Arcueid/Day 5/05_00_ARC05_2_2.txt [hunk 17]
```diff
@@ -167,22 +167,22 @@ But still――― Even a person like me can say what I don't want to do.
...
 -- Page 213, Offset 9125.
 -- ……だからいまさら、自分だけ逃げるなんて、できない」
-...That's why I can't run away by myself after all this."
+...So I can't just run away all by myself. Not after all this."
 }
 [sha:0f5e2a85185e13d39f5fc9b19dcd645e1c1f265f]{
 -- Page 214, Offset 9126.
 -- はあ、と天井を見上げて、大きく息を吸いこむ。
-I look up at the ceiling and take a deep breath.
+I raise my gaze to the ceiling and take a deep breath.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_2_2.txt:185]: Prefer simpler here tbh

File: script/Arcueid/Day 5/05_00_ARC05_2_2.txt [hunk 18]
```diff
@@ -197,15 +197,15 @@ My resolve strengthens as a result.
 [sha:e9f08a3949642f8396d24b0558a474fa1abf7cbf]{
 -- Page 215, Offset 9129. Glued.
 -- それがあのホテルにいて自分だけ生き残った、俺が取るべき責任なんだと思う」
- I believe this is the responsibility I need to take as the only survivor from that hotel."
+I believe this is my responsibility as the sole survivor of that tragedy at the hotel."
```
Comment (rschlaikjer) [inline script/Arcueid/Day 5/05_00_ARC05_2_2.txt:200]: Dropped leading space?

Comment (nolo16) [reply script/Arcueid/Day 5/05_00_ARC05_2_2.txt:200]: think i accidentally got rid of it :aichanteehee:

**Pull request merged**